apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/shard/ShardPrunableZipHashCalculator.java

```diff
@@ -78,21 +78,26 @@ private void recalculatePrunableArchiveHashes() {
                 ShardNameHelper shardNameHelper = new ShardNameHelper();
                 String prunableArchiveName = shardNameHelper.getPrunableShardArchiveNameByShardId(shard.getShardId(), chainId );
                 Path prunableArchivePath = dirProvider.getDataExportDir().resolve(prunableArchiveName);
+                ChunkedFileOps ops;
                 if (count == 0) {
                     shard.setPrunableZipHash(null);
                     shardDao.updateShard(shard);
                     Files.deleteIfExists(prunableArchivePath);
+                    ops = new ChunkedFileOps("");
                 } else {
                     String zipName = "shard-" + shard.getShardId() + ".zip";
-                    ChunkedFileOps ops = zip.compressAndHash(tempDirectory.resolve(zipName).toAbsolutePath().toString(), tempDirectory.toAbsolutePath().toString(), 0L, null, false);
-                    fileChangedEvent.select(new AnnotationLiteral<FileChangedEvent>(){}).fireAsync(ops);
-                    log.debug("Firing 'FILE_CHANDED' event {}", ops.getFileId());
+                    ops = zip.compressAndHash(tempDirectory.resolve(zipName).toAbsolutePath().toString(), tempDirectory.toAbsolutePath().toString(), 0L, null, false);
```

Comment:
Does variable  `String zipName = "shard-" + shard.getShardId() + ".zip";`
get correct value here ??
How that will process : `zip.compressAndHash(tempDirectory.resolve(zipName)` with value like = 'shard-1.zip' ??

Reply:
this is just temp file

Reply:
May be better to give it name like 'shard-temp-....' ?